pkg/scheduler/algorithm/priorities/metadata.go

```diff
@@ -32,7 +32,7 @@ type PriorityMetadataFactory struct {
 	statefulSetLister algorithm.StatefulSetLister
```

Comment:
What is above `log` file for?
Should remove such un-used files.

Reply:
Oh，I will remove it, thx!